libpod/util.go

```diff
@@ -369,3 +372,45 @@ func writeStringToPath(path, contents, mountLabel string, uid, gid int) error {
 
 	return nil
 }
+
+// getDefaultUlimits returns the default ulimits set based on whether podman is invoked
+// as rootless or root. This is used when determining what annotations to add for ulimits
+// during kube generate
+func getDefaultUlimits(limits []string) ([]string, error) {
```

Comment:
please add comments that this is the reverse implementation from addRlimits() and must be kept in sync

Reply:
I moved the code around a bit so we are not duplicating implementation of grabbing the default ulimit list.